### PR TITLE
[Landcover Toolkit] Set up continuous integration

### LINKED ISSUE #63: [Landcover Toolkit] Implement continuous integration in GitHub

### PR DESCRIPTION
**Pull request by gino-m**
- [x] Fixed remaining lint errors and rules
- [x] For PRs, only tests for modified packages (`tutorials`, `toolkits/landcover`)
- [x] Only run integration tests if keys avail (e.g., PR from same repo or cron job)
- [x] Configured Travis CI to run tests for changed packages on PRs and commits, and to run all tests (incl. integration tests) on daily cron job.

Example run:
https://travis-ci.org/google/earthengine-community/builds/598744139

@tylere FYI

Closes #63

### CODE REVIEW

**Comment by gino-m** [comment]
@tylere or @gorelick-google, PTAL. Thanks!

File: toolkits/landcover/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# limitations under the License.
+
+# Exit on error.
+set -e
+
+DIR="toolkits/landcover"
+KEY_FILE_ENC="test/.private-key.json.enc"
+KEY_FILE="test/.private-key.json.enc"
+KEY="${encrypted_44af26ab0da9_key}"
+IV="${encrypted_44af26ab0da9_iv}"
```
Comment (tylere) [inline toolkits/landcover/run-tests.sh]: Where do these strange looking variables come from?

File: toolkits/landcover/run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit on error.
+set -e
+
+DIR="toolkits/landcover"
+KEY_FILE_ENC="test/.private-key.json.enc"
```
Comment (tylere) [inline toolkits/landcover/run-tests.sh]: Why is toolkits/landcover/test/.private-key.json.enc being committed to the repo?

File: toolkits/landcover/run-tests.sh (hunk 1 above)
Comment (gino-m) [reply toolkits/landcover/run-tests.sh]: They're generated by Travis tools; Travis holds these keys used for unencrypting the service account's private key.

File: toolkits/landcover/run-tests.sh (hunk 2 above)
Comment (gino-m) [reply toolkits/landcover/run-tests.sh]: It's needed for Travis to run integration tests using our service account. Only Travis has the private key needed to unencode the key.

**Comment by gino-m** [comment]
@tylere All checks pass, please take another look!

**Pull request merged**